themes/digital.gov/src/scss/new/shortcodes/_do-dont-table.scss

```diff
@@ -0,0 +1,28 @@
+@use "uswds-core" as *;
+
+.dg-do-dont-table {
+  &__header--do {
+    @include u-border-top("05", "green-40v", "solid", "!important");
+    @include u-color("green-50v", "!important");
+  }
+
+  &__header--dont {
+    @include u-border-top("05", "red-50v", "solid", "!important");
+    @include u-color("red-50v", "!important");
+  }
+
+  &-row-do {
+    @include u-border-top("05", "green-40v", "solid");
+    @include u-color("green-50v");
+  }
```

Comment:
What element is this `dg-do-dont-table-row-do`?

Reply:
That should be removed.